lib/avalon/batch/entry.rb

```diff
@@ -160,8 +160,10 @@ def process!
           master_file.set_workflow(file_spec[:skip_transcoding] ? 'skip_transcoding' : nil)
           save_tries = 0
           begin
-            saved = media_object.save
+            media_object.save
+            saved = true
```

Comment:
Does this change mean this line https://github.com/ualbertalib/avalon/blob/d4a71dc56c5d15518881f6f99203bd4655ba25e9/lib/avalon/batch/entry.rb#L173 never worked before?  @cwant